docs/github/create-dotnet-github-action.md

```diff
@@ -0,0 +1,194 @@
+---
```

Comment:
I think we want to add a new section for "DevOps". This is likely the first article of a series that helps .NET developers use modern devops techniques for improved quality and velocity.

Reply:
Done